docs/runtime.md

```diff
@@ -151,6 +151,10 @@ After editing the configuration, restart `k0s` to get containerd using the newly
 
 Use the option `--cri-socket` to run a k0s worker with a custom CRI runtime. the option takes input in the form of `<type>:<socket_path>` (for `type`, use `docker` for a pure Docker setup and `remote` for anything else).
 
-To run k0s with a pre-existing Docker setup, run the worker with `k0s worker --cri-socket docker:unix:///var/run/docker.sock <token>`.
+To run k0s with a pre-existing Docker setup, run the worker with `k0s worker --cri-socket docker:unix:///var/run/cri-dockerd.sock <token>`.
```

Comment:
User also needs to setup the cri-dockerd shim, maybe just link to that page here?